templates/database/database-ss.yaml

```diff
@@ -50,6 +50,20 @@ spec:
 {{- if .Values.database.internal.initContainer.migrator.resources }}
         resources:
 {{ toYaml .Values.database.internal.initContainer.migrator.resources | indent 10 }}
+{{- end }}
+        volumeMounts:
+          - name: database-data
+            mountPath: /var/lib/postgresql/data
+            subPath: {{ $database.subPath }}
+      # if the volumn group is an NFS mount-point we need to ensure the postgres user will own the data directory. command
+      - name: "data-ownership-ensurer"
+        image: {{ .Values.database.internal.image.repository }}:{{ .Values.database.internal.image.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        command: ["/bin/sh"]
+        args: ["-c", "chown -R 999:999 /var/lib/postgresql/data || true"]
+{{- if .Values.database.internal.initContainer.permissions.resources }}
```

Comment:
this `.Values.database.internal.initContainer.permissions.resources` should probably be `.Values.database.internal.initContainer.ownership.resources`, and add it to values.yaml file